procrastinate/aiopg_connector.py

```diff
@@ -18,70 +18,23 @@
 class PostgresConnector(connector.BaseConnector):
     def __init__(
         self,
-        pool: aiopg.Pool,
+        *,
         json_dumps: Optional[Callable] = None,
         json_loads: Optional[Callable] = None,
+        **kwargs: Any,
     ):
         """
-        The pool connections are expected to have jsonb adapters.
+        Create a PostgreSQL connector. The connector is based on an
```

Comment:
"is based on", or "needs"?

Reply:
I decided to use "uses" here, with other improvements to the docstrings. See 537ee70c9323017a500cca49b793b6d1006bac48.